scripts/lint_jjb.py

```diff
@@ -225,6 +229,57 @@ def check_timed_trigger_value(schedule, data):
     return return_value
 
 
+# Routine to flatten lists, for example:
+# [[0, 1, 2],[[3, 4],[5, 6]], 7, 8] => [0,1,2,3,4,5,6,7,8]
+def flatten(s):
```

Comment:
You probably discovered that jmespath doesn't have a recursive flatten function. This is a bit more elegant than putting `[][][][][][][][][]` on the end of the expression.

Reply:
Yeah, I couldn't figure out how to flatten otherwise.